XO.py

```diff
@@ -0,0 +1,157 @@
+from random import randint
+from collections import namedtuple
+
+# Size of board.
+N = 3
+
+
+class Model:
+    def __init__(self):
+        self.board = [[None for x in range(N)] for y in range(N)]
+
+    def get_board_status(self):
+        return self.board
+
+    def update_board(self, line, col, players, curr_player):
+        if self.board[line][col] is not None:
+            return False
+        else:
+            self.board[line][col] = players[curr_player].symbol

```

Comment:
There's still a little mix between the model and the view here.
You're using symbol in the model, even though it's not the easiest solution, and it's done only for view purposes.
Why not use the player index instead, and translate later in the view layer?

Reply:
Got it :) Done